### ISSUE TITLE
sirenia cluster in unhealthy state (too few asyncs)

### ISSUE THREAD

**Issue by SalvoLunar** [flynn/flynn#4506]
My cluster collapsed earlier this morning.

As usual the problem seems to be mariadb, and it seems to be related to a memory issue.

I've restored the cluster from a backup reinstalling flynn on all nodes.

I've checked the limit for mariadb and i've noticed that they are the default ones:

```
mariadb:  cpu=1000  max_fd=10000  memory=1GB  temp_disk=100MB
web:      cpu=1000  max_fd=10000  memory=1GB  temp_disk=100MB
```
I've tried to change those values but I'm getting this error

`sirenia cluster in unhealthy state (too few asyncs)
`
so basically I have a cluster running and working fine but I cannot change those limits and therefore I assume it will soon go down again.

what can I do to fix this?

**Comment by philiplb** [comment]
It would be so awesome if these MariaDB problems could be fixed. They are the main reason we have to restore the cluster from backup just because a node rebooted.

**Comment by SalvoLunar** [comment]
the weird thing is, the cluster is working fine, if I check the status page all services are marked as healthy.

I've also checked that the controller_key is correct that the auth_key is correct and all seems to be in order, but when I try to change the limits for mariadb I still get the 
`sirenia cluster in unhealthy state (too few asyncs) ` error

I've restored the cluster this morning again and I cannot keep doing that every morning.
If anyone here has any suggestion on how to fix this please let me know.

**Comment by titanous** [comment]
Flynn is unmaintained and our infrastructure will shut down on June 1, 2021. See [the README](https://github.com/flynn/flynn#flynn-is-unmaintained) for details.
